src/OpenTelemetry.Instrumentation.Wcf/Implementation/WcfOperationContext.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using System.ServiceModel;
+
+namespace OpenTelemetry.Instrumentation.Wcf.Implementation;
+
+internal class WcfOperationContext : IExtension<OperationContext>
+{
+    public WcfOperationContext(Activity activity)
+    {
+        this.Activity = activity;
+    }
+
+    public Activity Activity { get; }
+
+    public void Attach(OperationContext owner)
+    {
+    }
+
+    public void Detach(OperationContext owner)
+    {
+    }
+}
```

Comment:
I hope I do not overlooked anything, could you please decorate it with conditional compilation?

```suggestion

#if NETFRAMEWORK
using System.Diagnostics;
using System.ServiceModel;

namespace OpenTelemetry.Instrumentation.Wcf.Implementation;

internal class WcfOperationContext : IExtension<OperationContext>
{
    public WcfOperationContext(Activity activity)
    {
        this.Activity = activity;
    }

    public Activity Activity { get; }

    public void Attach(OperationContext owner)
    {
    }

    public void Detach(OperationContext owner)
    {
    }
}
#endif
```